egs/wsj/s5/utils/split_data.sh

```diff
@@ -67,6 +67,11 @@ if [ -f $data/text ] && [ $nu -ne $nt ]; then
   echo "** use utils/fix_data_dir.sh to fix this."
 fi
 
+ns=`cat $data/spk2utt | wc -l`
+if [ $numsplit -gt $ns ]; then
+    echo "You should reduce the number of jobs ($numsplit) as there are not enough speakers ($ns)."
```

Comment:
don't do this if the --per-utt option is given

Reply:
fixed